packages/lsp/src/connection_manager.ts

```diff
@@ -653,7 +652,7 @@ namespace Private {
     capabilities: ClientCapabilities
   ): Promise<LSPConnection> {
     let connection = _connections.get(languageServerId);
-    if (connection == null) {
+    if (connection === undefined) {
```

Comment:
```suggestion
    if (connection) {
```

Reply:
I changed it to `if (!connection)` since that's the condition I was replacing.

Reply:
Ah sorry i misread the condition.